test/test_nestedtensor.py

```diff
@@ -8495,6 +8567,18 @@ def __torch_dispatch__(self, func, types, args=..., kwargs=None):
         op_match_fn=lambda device, op: (op.full_name in {"cdouble", "cfloat", "chalf"}),
         name="unimplemented_view_as_real",
     ),
+    # narrow(): unbacked SymInt bug with non-contig transposed inputs
```

Comment:
Oof. Just curious - Is there an issue somewhere? If there is, maybe worth linking.